src/main/java/com/shepherdjerred/capstone/server/events/handlers/PlayerJoinEventHandler.java

```diff
@@ -1,11 +1,21 @@
 package com.shepherdjerred.capstone.server.events.handlers;
 
 import com.shepherdjerred.capstone.events.handlers.EventHandler;
+import com.shepherdjerred.capstone.network.packet.packets.PlayerDescriptionPacket;
+import com.shepherdjerred.capstone.server.GameServer;
 import com.shepherdjerred.capstone.server.events.events.PlayerJoinEvent;
+import com.shepherdjerred.capstone.server.network.ConnectorHub;
+import lombok.AllArgsConstructor;
 
+@AllArgsConstructor
 public class PlayerJoinEventHandler implements EventHandler<PlayerJoinEvent> {
+  private final GameServer gameServer;
+  private final ConnectorHub connectorHub;
 
   @Override
-  public void handle(PlayerJoinEvent event) {
+  public void handle(PlayerJoinEvent playerJoinEvent) {
+    gameServer.addPlayer(playerJoinEvent.getClientId(), playerJoinEvent.getPlayer());
+    connectorHub.sendPacket(new PlayerDescriptionPacket(playerJoinEvent.getPlayer().getName()) {
```

Comment:
Don't see a player description packet, create a new packet type called PlayerJoined (or something similar). Our clients need the entire player object, not just the name.